Library/Formula/wine.rb

```diff
@@ -116,7 +116,6 @@ def install
     ENV.libxml2
 
     args = ["--prefix=#{prefix}"]
-    args << "--disable-win16" if MacOS.version <= :leopard
```

Comment:
Flip the sign?
`if MacOS.version <= :leopard`
means leopard, tiger, panther, etc
or leopard, snow leopard, lion, mountain lion, mavericks, etc
I would think it's the latter.